lib/ui/view/channel/edit_channel/edit_channel_view_model.dart

```diff
@@ -34,28 +34,29 @@ class EditChannelViewModel extends FormViewModel {
       snackbar.showCustomSnackBar(
         duration: const Duration(seconds: 3),
         variant: SnackbarType.failure,
-        message: fillAllFields,
+        message: 'Please fill all fields.',
       );
 
       return;
     }
     const channel_id = '613f70bd6173056af01b4aba';
```

Comment:
Don't use hard coded values.
get the org id from local storage.
pass the channel id through the function parameters